ArkEcosystem/explorer
# refactor: show known wallet name when defined

## Summary

https://app.clickup.com/t/1p633dx

Updates the cached delegate username command to:
1. Cache the know wallet name when it is defined 
2. Refactors the command to cache all usernames into a single task instead of relaying on queued jobs)
3. Remove deprecated job for caching usernames

Since the known wallet name will be saved that what we are going to show on the tables.

### To test:
The know wallets names come from an URL defined on the settings (`config/explorer.php`) so you have a couple of options:

1. Work with a production network `EXPLORER_NETWORK=production` but you will ned to have a address in your database that matches any of the know wallet names from here `https://raw.githubusercontent.com/ArkEcosystem/common/master/mainnet/known-wallets-extended.json`
2. You can create a custom `known-wallets.json` file with some addresses that match the ones you have in your local environment, mount a local server with that JSON file and add the `knownWallets` setting on the config (hope you know how to mount a local server 😂)
3. You can use your years of experience to read the code and assume its working (not recommended)
4. After you decide just run `php artisan explorer:cache-delegate-usernames` and look for a table of transactions that contains the address in question, you should see the known wallet name


## Checklist
